docker/builder/Dockerfile

```diff
@@ -68,24 +68,24 @@ RUN DEBIAN_FRONTEND="noninteractive" apt-get install -y --no-install-recommends
         mcpp \
         sqlite3 \
         zlib1g-dev && \
-    wget -nv https://github.com/souffle-lang/souffle/archive/refs/tags/2.1.tar.gz \
-        -O /build/souffle-2.1.tar.gz && \
-    tar xf souffle-2.1.tar.gz && \
-    cd souffle-2.1 && \
+    wget -nv https://github.com/souffle-lang/souffle/archive/refs/tags/2.4.tar.gz \
+        -O /build/souffle-2.4.tar.gz && \
+    tar xf souffle-2.4.tar.gz && \
+    cd souffle-2.4 && \
     cmake -S . -B build && \
     cmake --build build --target install DESTDIR=/staging/full && \
     cd /build && \
-    rm -rf souffle-2.1 souffle-2.1.tar.gz
+    rm -rf souffle-2.4 souffle-2.4.tar.gz
 
 # Compile Fastobo-validator.
-RUN wget -nv https://github.com/fastobo/fastobo-validator/archive/refs/tags/v0.4.0.tar.gz \
-        -O /build/fastobo-validator-0.4.0.tar.gz && \
-    tar xf fastobo-validator-0.4.0.tar.gz && \
-    cd fastobo-validator-0.4.0 && \
+RUN wget -nv https://github.com/fastobo/fastobo-validator/archive/refs/tags/v0.4.6.tar.gz \
+        -O /build/fastobo-validator-0.4.6.tar.gz && \
+    tar xf fastobo-validator-0.4.6.tar.gz && \
```

Comment:
Did I dream this or didnt you replace all of these with a Variable in an earlier PR?

Reply:
I did, but I didn’t cherry-pick _that_ change here. I’d rather keep all changes in the maintenance branch to the strict minimum.

The change will be present in the future 1.5 maintenance branch.

Reply:
Merging that to the 1.4 maintenance branch now. After that, as discussed on Slack the ball is in your hands to decide when to make a release out of that branch.